dependencyManagement/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ val byteBuddyVersion = "1.12.19"
 val jmhVersion = "1.36"
 val mockitoVersion = "4.9.0"
 val slf4jVersion = "2.0.4"
+val otelContribVersion = "1.20.0"
```

Comment:
this looks unused?

Reply:
Oops yeah planned on referencing it with `io.opentelemetry.contrib:opentelemetry-aws-xray-propagator`. Will remove for now until we add dependencies on more contrib artifacts.